optuna/visualization.py

```diff
@@ -92,6 +93,114 @@ def _get_intermediate_plot(study):
     return figure
 
 
+def plot_parallel_coordinate(study, params=[]):
+    # type: (Study, List[str]) -> None
+    """Inside Jupyter notebook, plot the high-dimentional parameter relationships in a study.
```

Comment:
I think https://github.com/pfnet/optuna/pull/513#discussion_r326118285 is also applied to this document.